importer/src/main/java/org/ihtsdo/rvf/importer/AssertionGroupImporter.java

```diff
@@ -348,6 +348,7 @@ public boolean isImportRequired() {
 	 * file-centric-validation
 	 * release-type-validation
 	 * component-centric-validation
+	 * delta-file-required
```

Comment:
@QuyenLy87 As discussed in the call, we don't need to create a new assertion group for delta-file-required.

Reply:
Done